src/NetAnalyzers/UnitTests/Microsoft.NetCore.Analyzers/Runtime/InstantiateArgumentExceptionsCorrectlyTests.cs

```diff
@@ -915,6 +915,46 @@ End Sub
                End Class");
         }
 
+        [Fact]
+        public async Task ArgumentExceptionType_NotHavingConstructorWithParameterName_NoArgument_DoesNotWarn()
+        {
+            await VerifyCS.VerifyAnalyzerAsync(@"
+                public class Class
+                {
+                    public void Test(string first)
+                    {
+                        throw new System.Text.DecoderFallbackException ();
+                    }
+                }");
+
+            await VerifyVB.VerifyAnalyzerAsync(@"
+               Public Class [MyClass]
+                   Public Sub Test(first As String)
+                       Throw New System.Text.DecoderFallbackException ()
+                   End Sub
+               End Class");
+        }
+
+        [Fact]
+        public async Task ArgumentExceptionType_NotHavingConstructor_WithParameterName_WithArgument_DoesNotWarn()
+        {
+            await VerifyCS.VerifyAnalyzerAsync(@"
+                public class Class
+                {
+                    public void Test(string first)
+                    {
+                        throw new System.Text.DecoderFallbackException (""first"");
```

Comment:
One of the exceptions derived from ArgumentException but not have a constructor with `paramName` argument